python/setup.py.in

```diff
@@ -60,13 +60,14 @@ ext_modules = [
                 'cxx/summary_state.cpp',
                 'cxx/table_manager.cpp',
                 'cxx/well.cpp',
-                'cxx/export.cpp'
+                'cxx/export.cpp',
+                'cxx/builtin_pybind11.cpp'
```

Comment:
It is probably me, but I cannot find this file (either here or in master)-

Reply:
It (`cxx/builtin_pybind11.cpp`) is [automatically generated](https://github.com/OPM/opm-common/blob/877f82ef06bf186be0d6e1fd902af032fb01495b/src/opm/parser/eclipse/Parser/createDefaultKeywordList.cpp#L65-L66) if you [enable Python support](https://github.com/OPM/opm-common/blob/877f82ef06bf186be0d6e1fd902af032fb01495b/GenerateKeywords.cmake#L113-L117).

Reply:
Thanks. looks good to me then.